src/resources/js/components/AccountCreation.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div>
+    <p class="title">PAUL</p>
+    <form @submit.prevent="register" method="post">
+      <div class="form-group">
+        <input type="text" class="input" name="lname" v-model="input.lname" placeholder="Nom" />
+      </div>
+      <div class="form-group">
+        <input type="text" class="input" name="fname" v-model="input.fname" placeholder="Prénom" />
+      </div>
+      <div class="form-group">
+        <input type="email" class="input" name="email" v-model="input.email" placeholder="email" />
+      </div>
+      <div class="form-group">
+        <input type="password" class="input" name="password" v-model="input.password" placeholder="Mot de passe" />
+      </div>
+      <div class="form-group">
+        <input type="password" class="input" name="passwordConfirm" v-model="input.passwordConfirm" placeholder="Confirmer mot de passe" />
+      </div>
+      <div class="form-group">
+        <button type="button" class="button" v-on:click="register()">Créer compte</button>
+      </div>
+      
+      <p v-if="isBlank"> Manque des informations</p>
+      <p v-if="isIncorect"> mot de passe différent</p>
+    </form>
+  </div>
+</template>
+
+<script>
+    import axios from 'axios'
+    import router from '../routes'
+
+    export default {
+    name : 'Register',
+    data () {
+        return{
+          input: {
+              lname: '',
+              fname: '',
+              email: '',
+              password: '',
+              passwordConfirm: ''
+          },
+      isBlank: false,
+      isIncorect: false}
```

Comment:
I think one message flag and one message string is better than multiple flags and multiple messages as it's easier to understand (like in #34 and #33 )
```
      isMessage: false,
      message: '',
```

Reply:
Changed in 313adfc